src/classes/CW_CampaignRewardCompController.cls

```diff
@@ -0,0 +1,668 @@
+/**
+* File Name   :    CW_CampaignRewardCompController.cls
+* Description :    B-109691 - Reward Summary Table 
+*				   that are linked to the campaign 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 07/05/2014 Alex Panka	  Intial  
+* 0.2 08/05/2014 Chris Yorkston added method updateRewardWithFreebetId() to 
+*                               retreive Freebet Id from Apex Callout (HTTP)
+* 0.3 20/07/2014 Cristina P.	B-122940 Campaign Reward - Many Rewards per Trigger
+*/
+ 
+public with sharing class CW_CampaignRewardCompController {
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.